src/fhir/guides/ig-atf/Home/Einfuehrung.page.md

```diff
@@ -1,41 +1,76 @@
+
+Im MessageHeader verweist das Feld `.focus` auf eine oder mehrere Ressourcen, die den jeweiligen Anwendungsfall betreffen. Die genaue Definition variiert je nach Projekt, das diese Spezifikation umsetzt.
 
 ### Verwendung von MessageHeader.eventCode
 
-Historisch ist MessageHeader.eventCode ein wesentlicher Bezeichner, der angibt, wie das empfangene System die Nachricht zu verarbeiten hat. Derzeit entsprechen der .eventCode den KIM-Dienstkennungen, die auch in der Umgebenden KIM-Nachricht mit angegeben werden.
+Der **MessageHeader.eventCode** spielt eine zentrale Rolle bei der Verarbeitung der Nachricht im empfangenden System. Er gibt an, welcher Anwendungsfall behandelt wird und über welchen Übertragungsweg dies erfolgt. Aktuell orientieren sich die eventCodes an den KIM-Dienstkennungen, die ebenfalls in der umgebenden KIM-Nachricht zu finden sind.
 
-Der EventCode drückt aus, welcher Anwendungsfall und welcher Übertragungsweg innerhalb eines Anwendungsfalls behandelt wird.
+Der **eventCode** definiert somit nicht nur den Anwendungsfall, sondern auch den Übertragungsweg innerhalb eines Anwendungsfalls.
 
 ## Einordnung und Abgrenzung
 
-Diese Spezifikation beschreibt den umgebenden Container (MessageBundle) und einen Eintrag in diesem Bundle, der immer vorhanden sein muss. Die Anwendungsfälle, die von diesem Projekt abhängen werden möglicherweise weitere Einschränkungen am Bundle und MessageHeader vornehmen, aber das Kommunikationspattern übernehmen.
+Diese Spezifikation definiert den übergeordneten Container (**MessageBundle**) und einen festen Bestandteil dieses Containers, den **MessageHeader**. Während spezifische Anwendungsfälle zusätzliche Einschränkungen oder Anforderungen an das Bundle und den MessageHeader haben können, bleibt das grundlegende Kommunikationsmuster unverändert.
 
 Jede Nachricht besteht also aus einem Bundle, welches mindestens den MessageHeader und je nach Anwendungsfall weitere Ressourcen enthält.
 
-{{render:src/fhir/guides/ig-atf/images/ATF-Scope.png}}
+## Stufenweise Einführung des App-Transport-Frameworks
+
+Die Implementierung des App-Transport-Frameworks erfolgt schrittweise, um den reibungslosen Übergang und die schrittweise Optimierung der Kommunikation im medizinischen Umfeld zu gewährleisten.
+
+### Stufe 1: Standardisierung und Zuverlässigkeit
+
+Die erste Stufe des App-Transport-Frameworks (ATF) etabliert wichtige Grundlagen für die strukturierte und zuverlässige Kommunikation im medizinischen Bereich. Sie führt folgende wesentliche Verbesserungen ein:
+
+- **Standardisierung und Klarheit**: Die klare **Use-Case-Definition** über das Codesystem des Anwendungsfalls sorgt für eine präzise und einheitliche Struktur beim Austausch von medizinischen Daten. Dies ermöglicht eine nahtlose Interoperabilität zwischen verschiedenen Systemen und Anwendungen, indem jeder Anwendungsfall klar definiert wird.
```

Comment:
Mir sind beim Lesen die Inhalte der Stufe nicht ganz klar. Ich würde die Funktion der Stufe vorne in Dick schreiben, also z.B.
Empfangsbestätigung: Für Sicherheit und Effizienz...lalala
Use-Case-Definitionen: lalala
usw.

Reply:
Durch das GPT ist es für mich auch irgendwie ulkig zu lesen, also ich weiß nicht ganz was damit gemeint ist

Reply:
Das Dick schreiben der KeyWords ist doch schon der Fall. 
Was soll konkret geändert/verbessert werden?